static/js/scroll_util.js

```diff
@@ -61,25 +61,28 @@ export function scroll_element_into_container_for_buddy_list(elem, container) {
 
     const delta = scroll_delta(opts);
 
-    // function do_elements_overlap (rect1, rect2) {
-    //     return !(rect1.top > rect2.bottom);
-    // }
+    function do_elements_overlap(rect1, rect2) {
+        return !(rect1.top > rect2.bottom);
+    }
 
     if (delta === 0) {
-        // if ($("#users_heading")[0]) {
-        //     const rect1 = elem[0].getBoundingClientRect();
-        //     const rect2 = $("#users_heading")[0].getBoundingClientRect();
-        //     if (do_elements_overlap(rect1, rect2)) {
-        //         container.scrollTop(container.scrollTop() - (rect2.bottom - rect1.top));
-        //     }
-        // }
-        // if (buddy_list.other_keys && buddy_list.other_keys.includes(buddy_list.get_key_from_row({row: elem}))) {
-        //     const rect1 = elem[0].getBoundingClientRect();
-        //     const rect2 = $("#others_heading")[0].getBoundingClientRect();
-        //     if (do_elements_overlap(rect1, rect2)) {
-        //         container.scrollTop(container.scrollTop() - (rect2.bottom - rect1.top));
-        //     }
-        // }
+        if ($("#users_heading")[0]) {
```

Comment:
Bit confused about the code that was commented in [hack: Add ListCursorWithScrollOverride hack.](https://github.com/zulip/zulip/pull/20072/commits/7ae206b1ce28cfe3b2e7a1bf74f45e7fb3f2758a), is uncommented here again. 

@YashRE42 is this because of rebased?